app/Model/Attribute.php

```diff
@@ -4077,6 +4077,12 @@ public function editAttribute($attribute, $eventId, $user, $objectId, $log = fal
                 $attribute['distribution'] = 5;
             }
         }
+        if (isset($attribute['Sighting']) && !empty($attribute['Sighting'])) {
+            $this->Sighting = ClassRegistry::init('Sighting');
+            foreach ($attribute['Sighting'] as $s) {
```

Comment:
You should use `Sightings::captureSightings` method.

Example usage:
https://github.com/MISP/MISP/blob/bd98ea937de43b8a311b1f71912ba71d3e674746/app/Model/Attribute.php#L3977-L3979

Reply:
Thanks @JakubOnderka, didn't spot that was an available function, will update the code :)